modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Update the proxy settings used by Runtime Fabric outbound connections to the internet. This includes managing HTTP proxies for connecting out to the Anypoint control plane, and SOCKS5 proxies for sending metrics and logs out to Anypoint Monitoring.
+You can update the proxy settings used by Anypoint Runtime Fabric outbound connections to the internet. This includes managing both HTTP proxies for connecting to the Anypoint Platform control plane and SOCKS5 proxies for sending metrics and logs to Anypoint Monitoring.
```

Comment:
This reads as if "update" and "managing" are in the wrong order?
```suggestion
You can manage the proxy settings used by Anypoint Runtime Fabric outbound connections to the internet. This includes updating both HTTP proxies for connecting to the Anypoint Platform control plane and SOCKS5 proxies for sending metrics and logs to Anypoint Monitoring.
```
"updating" is part of "managing," but I don't think that the reverse is precisely true.